shared/src/interface_config.rs

```diff
@@ -50,6 +50,11 @@ pub struct ServerInfo {
 
     /// An internal endpoint in the WireGuard network that hosts the coordination API.
     pub internal_endpoint: SocketAddr,
+
+    /// Whether the routing should be done by innernet or is done by an
+    /// external tool like e.g. babeld.
+    /// Analog to what wg-quick does with "Table = off" if true.
+    pub external_routing: bool,
```

Comment:
It was not clear to me where the line is drawn between `ServerInfo` and `InterfaceInfo`. I added the `external_routing` to `ServerInfo` because that is what is already available in `add_peer` in `client/src/main.rs`.